Src/Base/AMReX_GpuDevice.cpp

```diff
@@ -22,9 +22,9 @@
 #if defined(AMREX_USE_HIP)
 #include <hip/hip_runtime.h>
 #if defined(AMREX_USE_ROCTX)
-#include <roctracer_ext.h>
```

Comment:
If you make this change, please update `Tools/GNUMake/comps/hip.mak`.

Reply:
Good idea, done.